gatsby-node.js

```diff
@@ -1,5 +1,28 @@
 const path = require('path');
 const { createFilePath } = require('gatsby-source-filesystem');
+const axios = require('axios')
+
+// exports.sourceNodes = async ({ actions, reporter, createContentDigest }) => {
```

Comment:
My idea here is to use this instead of the airtable plugin, so we don't need to have the Airtable API key for local development. But I don't know where it should point to (always point to the "prod" deployed function?).

Reply:
On our scale now I think pointing to `prod` is ok